package.json

```diff
@@ -33,7 +33,6 @@
   "homepage": "https://github.com/wikimedia/restbase-mod-table-spec",
   "dependencies": {
     "routeswitch": "~0.6.3",
-    "cassandra-uuid": "^0.1.0",
```

Comment:
Em.. We kinda need to depend on `uuid` to use it..

Reply:
I run a check now to avoid in the future